hack/e2e-common.sh

```diff
@@ -13,7 +13,7 @@ set -o monitor
 # Compute the timeout as 2m less than what's configured for the job in prow --
 # keep these up to date with the job config!
 if [[ $0 == */e2e-pool-test.sh ]]; then
-  timeout_minutes=148
+  timeout_minutes=182
```

Comment:
```suggestion
  timeout_minutes=118
```
The idea is to make this *shorter* than the 2h overall timeout that seems to be hitting us for reasons we still can't figure out.

Reply:
or just wait for https://github.com/openshift/hive/pull/1805 to merge, and rebase.

Reply:
brain fart